pdb2sql/interface.py

```diff
@@ -29,14 +29,32 @@ def get_contact_atoms(
             extend_to_residue=False,
             only_backbone_atoms=False,
             excludeH=False,
```

Comment:
there was a distinction here as to consider only the backbone atoms and return only the backbone atoms. I'm not sure that's correct here anymore. I have a test somewhere where I compare the results with PROFIT. We should make that part of the testing as well

Reply:
It's correct, the latest unit tests have included this part.